examples/gluon_alexnet/MANIFEST.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Why do we need to expose the manifest.json? This is an internal file that is created by MMS, not something the user should care about, right?
What happens if we change the manifest later, why would we need to go back and update manifest in examples?
I suggest we remove this file from the PR.

Reply:
+1
Also can you move the gluon examples under examples/gluon/alexnet.. gluon/hybrid/..

Reply:
Agreed. Modified Export tool to handle imperative models and updated the tests.